crates/pallets/starknet/Cargo.toml

```diff
@@ -30,6 +30,7 @@ frame-system = { workspace = true }
 frame-benchmarking = { workspace = true, optional = true }
 
 # Substrate primitives
+sp-api = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40" }
```

Comment:
we should probably add it to the parent `cargo.toml` like we do for the other crates

Reply:
moved to parent `cargo.toml`